respa_admin/templates/respa_admin/_nav.html

```diff
@@ -9,6 +9,14 @@
         <!-- Collect the nav links, forms, and other content for toggling -->
         <div class="collapse navbar-collapse">
             <ul class="nav navbar-nav navbar-right">
+                {% if INSTRUCTIONS_URL %}
+                <li>
+                    <a href="{{ INSTRUCTIONS_URL }}">
```

Comment:
Add target="_blank" here so the link opens in a new tab. Otherwise the user can lose unsaved data in the form if the link opens in the current tab.